site/en/docs/android/custom-tabs/browser-comparison/index.html

```diff
@@ -0,0 +1,1378 @@
+---json
+{
+  "layout": "layouts/doc-post.njk",
+  "title": "CustomTab browser comparison",
+  "seoTitle": "Guide: An overview of Custom Tab features, and their browser availability",
+  "description": "An overview of Custom Tab features, and their browser availability",
+  "authors": [
+    "patrickkettner"
+  ]
+}
+---
+<html><head>
```

Comment:
@matthiasrohmer are you aware of a better way to integrate custom tables than generating HTML code? This feels hard to maintain in the future.